### DISCUSSION TITLE
Why release.gitlab requires `api` scope token?

### DISCUSSION THREAD

**Discussion by mishak87** [goreleaser/goreleaser#2208] [Q&A] [answered]
GitLab CI_JOB_TOKEN is allowed to create a release. What other API calls goreleaser is using that require `api` scope?
https://docs.gitlab.com/ee/api/README.html#gitlab-cicd-job-token

**Comment by mavogel** [comment, marked as answer]
You might want to take a look here https://github.com/goreleaser/goreleaser/blob/master/internal/client/gitlab.go

**Reply by mishak87** [reply]
I assumed it only creates the release without detection/update logic. Would it be possible when https://github.com/goreleaser/goreleaser/issues/2006 is implemented to have option of create-only release?
My use-case is run goreleaser and create gitlab release. I am ok with updating/deleting bad releases manually if that is what it takes to not have "root" token in my pipeline.
Especially since https://gitlab.com/groups/gitlab-org/-/epics/3559 can take years to complete.

**Reply by mavogel** [reply]
TBH I do the same if a release failed or is incomplete. Initially we kept the gitlab release close the the Github one, which had the update possibility of a release in place. From my point of view we could move to `create-only`. @caarlos0 WDYT?

**Reply by caarlos0** [reply]
@mavogel I think it makes sense yes...
